.eslintrc.json

```diff
@@ -22,30 +22,30 @@
       "__DEV__": true
     },
     "rules": {
-      "global-require": 0,
-      "no-undefined": 0,
-      "react/display-name": [2, { "ignoreTranspilerName": false }],
-      "react/jsx-filename-extension": 0,
-      "camelcase": [
-        0,
-        {
-          "properties": "never"
-        }
-      ],
-      "@typescript-eslint/ban-types": 0,
-      "@typescript-eslint/no-non-null-assertion": 0,
-      "@typescript-eslint/no-unused-vars": [
-        2,
-        {
-          "vars": "all",
-          "args": "after-used"
-        }
-      ],
-      "@typescript-eslint/no-explicit-any": "warn",
-      "@typescript-eslint/no-use-before-define": 0,
-      "@typescript-eslint/no-var-requires": 0,
-      "@typescript-eslint/explicit-function-return-type": 0,
-      "@typescript-eslint/explicit-module-boundary-types": "off"
+      // "global-require": 0,
```

Comment:
why disable the rules?

Reply:
Most of these rules were actually disabling a lot of the recommend checks from the esling/typescript/mattermost-react rules, so throwing lots of warnings and impeding development. The PR sets up the types so we can adhere to all the recommended standards and not need to have rules disabling various checks / warnings.